packages/react-native-reanimated/Common/NativeView/react/renderer/components/rnreanimated/ComponentDescriptors.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <react/renderer/componentregistry/ComponentDescriptorProviderRegistry.h>
+#include <react/renderer/core/ConcreteComponentDescriptor.h>
+#include <react/renderer/components/rnreanimated/ReanimatedShadowNode.h>
+
+namespace facebook::react {
+
+using ReanimatedViewComponentDescriptor =
+    ConcreteComponentDescriptor<ReanimatedShadowNode>;
+
+void rnreanimated_registerComponentDescriptorsFromCodegen(
+    std::shared_ptr<const ComponentDescriptorProviderRegistry> registry);
```

Comment:
Can we use a const ref here?
```suggestion
    const std::shared_ptr<const ComponentDescriptorProviderRegistry> &registry);
```